src/basic/INIT_VIEW1D.cpp

```diff
@@ -36,8 +36,7 @@ namespace basic
   const Real_type v = m_val; \
 \
   using ViewType = RAJA::View<Real_type, RAJA::Layout<1> >; \
-  const RAJA::Layout<1> my_layout(iend); \
-  ViewType view(a, my_layout);
+  RAJA::View<Real_type, RAJA::Layout<1,RAJA::Index_type,0> > view(a,iend);
```

Comment:
How is this different than the original View definition for a 1d view?

Reply:
The '0' in the layout template specialization indicates that the 0th index is unit stride thus it does not get multiplied by stride. And yes, I can make the changes to the CUDA and OMP Target versions too.

Reply:
So, in the original version the index got multiplied by '1'? That's not what I would expect.